examples/http_crawl.cpp

```diff
@@ -46,10 +46,11 @@ int main(int, char const*[])
             stream<ip::tcp::socket> hs(ios);
             connect(hs.lowest_layer(), it);
             auto ep = hs.lowest_layer().remote_endpoint();
-            request<empty_body> req({method_t::http_get, "/", 11});
+            request<empty_body> req({"GET", "/", 11});
             req.headers.insert("Host", host +
                 std::string(":") + std::to_string(ep.port()));
             req.headers.insert("User-Agent", "beast/http");
+            prepare(req);
             hs.write(req);
             response<string_body> resp;
             hs.read(resp);
```

Comment:
I got this example to run.  I have no idea what it is trying to do, but it sure spent quite a while trying to do it!